### PR TITLE
Updated the README.md with information on how to use NUnit.Xamarin wi…

### PR DESCRIPTION
**Pull request by moritzh**
…th Xamarin Studio.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,4 +254,108 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
...
     // <SNIP>
 }
-```
\ No newline at end of file
+```
+## How to Use (Xamarin Studio)##
+
+If you're using Xamarin Studio the templates above won't work. This guide assumes that you have a solution with either a PCL or a Shared project and a number of platform specific projects. It doesn't matter if you're using Xamarin.Forms or Xamarin.iOS/Xamarin.Android directly. Your solution might look something like this:
+
+```C#
```
Comment (rprouse) [inline README.md]: Could you remove the C# language specification from all these code blocks? This isn't C#, so should just be displayed as a plain text <pre>.

**Comment by rprouse** [comment]
Looks good. The C# that I mentioned looks fine when viewed, it just bothers me because it isn't semantically correct. OCD kicking in :smile: 

If you can make that minor fix, I will merge.  Thanks for the contribution.

CI Build failures are my fault. I keep meaning to try getting it working in AppVeyor.

**Comment by moritzh** [comment]
Done!

**Comment by rprouse** [comment]
Looks good, merging.

**Pull request merged**